### PR TITLE
Update insider-risk-management-configure.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to mention that these role groups on PIM

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2941

File: microsoft-365/compliance/insider-risk-management-configure.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Depending on the structure of your compliance management team, you have options
 | **Insider Risk Management Analysts** | Use this group to assign permissions to users that will act as insider risk case analysts. Users in this role group can access all insider risk management alerts, cases, and notices templates. They cannot access the insider risk Content Explorer. |
 | **Insider Risk Management Investigators** | Use this group to assign permissions to users that will act as insider risk data investigators. Users in this role group can access all insider risk management alerts, cases, notices templates, and the Content Explorer. |
 
+>[!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-configure.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/compliance/insider-risk-management-configure.md [hunk 2]
```diff
@@ -51,6 +51,9 @@ Depending on the structure of your compliance management team, you have options
...
+>These Role groups are currently not supported on Privileged Identity Management (PIM). To Learn more about PIM, See [Azure AD roles in Privileged Identity Management](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-how-to-add-role-to-user)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/insider-risk-management-configure.md]:
```suggestion
> These role groups are currently not supported on Privileged Identity Management (PIM). To learn more about PIM, see [Assign Azure AD roles in Privileged Identity Management](https://docs.microsoft.com/azure/active-directory/privileged-identity-management/pim-how-to-add-role-to-user).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**